Modules/ModParser-3_0.lua

```diff
@@ -1669,6 +1669,7 @@ local specialModList = {
 	["you have tailwind if you have dealt a critical strike recently"] = { flag("Condition:Tailwind", { type = "Condition", var = "CritRecently" }) },
 	["your aura buffs do not affect allies"] = { flag("SelfAurasCannotAffectAllies") },
 	["aura buffs from skills have (%d+)%% increased effect on you for each herald affecting you"] = function(num) return { mod("AuraBuffEffect", "INC", num, { type = "Multiplier", var = "Herald"}) } end,
+	["aura buffs from skills have (%d+)%% increased effect on you for each herald affecting you, up to (%d+)%%"] = function(num, _, limit) return { mod("PurpHarbAuraBuffEffect", "INC", num, { type = "Multiplier", var = "Herald", limit = tonumber(limit),}) } end,
```

Comment:
Doesn't need the limit on it. Can just be 
`["aura buffs from skills have (%d+)%% increased effect on you for each herald affecting you, up to (%d+)%%"] = function(num) return { mod("PurpHarbAuraBuffEffect", "INC", num, { type = "Multiplier", var = "Herald"}) } end,`